tests/test_notification_system.py

```
@@ -0,0 +1,252 @@
+            NotifierCreate(
+                name="Test",
+                type=NotifierType.EMAIL,
+                address="test@example.com",
+                subscribed_events=["SCAN_COMPLETED", "INVALID_EVENT"],
+            )
+
+
+class TestXSSPrevention:
+    """Test XSS prevention in notification HTML."""
+
+    def test_html_message_escaping(self, db_session: Session):
```

Ruff (0.14.13)

Unused method argument: `db_session`

(ARG002)